packages/web-components/src/components/masthead/cloud/cloud-masthead.scss

```diff
@@ -110,3 +116,17 @@
     background-color: $hover-ui;
   }
 }
+
+:host(#{$dds-prefix}-cloud-left-nav-item) {
+  @extend :host(#{$dds-prefix}-left-nav-item);
+
+  &.left-nav-cta .bx--side-nav__link .bx--side-nav__link-text {
+    color: $link-01;
+  }
+}
+
+:host(#{$dds-prefix}-cloud-masthead-profile) .bx--header__menu-title[aria-expanded='true'] + .bx--header__menu {
+  @include carbon--breakpoint-down(320px) {
```

Comment:
Wondering for the profile menu - do we only want the menu to be full width when on 320px or smaller? I only see the menu going full width on the iphone 5/SE currently. Maybe it makes more sense to do:

``` 
@include carbon--breakpoint(sm) {
    width: 100vw;
  }
```

Reply:
Indeed. Not sure from the design docs at exactly which breakpoint the menu moves to full-width. I combined our ideas. :) https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/5856/commits/a5050a3e1387c45f7a8662a2adafe9eb70a4fcb8 Thanks!